### PR TITLE
added IE-2000-8TC-B

### PR DESCRIPTION
**Pull request by jacob-behnken**
pre-commit successful

### CODE REVIEW

File: device-types/Cisco/IE-2000-8TC-B.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    type: 100base-tx
+  - name: FastEthernet1/9
+    type: 100base-tx
+  - name: FastEthernet1/10
+    type: 100base-tx
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
```
Comment (danner26) [inline device-types/Cisco/IE-2000-8TC-B.yaml:33]: Is this used for console access? If not, please remove it from this file.

File: device-types/Cisco/IE-2000-8TC-B.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Cisco
+model: IE-2000-8TC-B
+part_number: IE-2000-8TC-B
+slug: cisco-ie-2000-8tc-b
+u_height: 3
+is_full_depth: false
```
[the remark this answers is not in the document]
Comment (jacob-behnken) [reply device-types/Cisco/IE-2000-8TC-B.yaml:7]: added

[the remark this answers is not in the document]
Comment (jacob-behnken) [reply device-types/Cisco/IE-2000-8TC-B.yaml:7]: DC power input on the chassis, external module required for AC to DC conversion.

File: device-types/Cisco/IE-2000-8TC-B.yaml (hunk 1 above)
Comment (jacob-behnken) [reply device-types/Cisco/IE-2000-8TC-B.yaml:35]: used for console

**Comment by jacob-behnken** [comment]
added weight. There is no airflow since its a solid state device. It also has DC power input on the chassis. A modular power supplly would b

**Comment by stuntguy3000** [comment]
@jacob-behnken `airflow: passive` is what you're looking for

**Comment by jacob-behnken** [comment]
thanks @stuntguy3000 its been added

**Comment by danner26** [comment]
Hello @jacob-behnken it does not look like you have added the DC power to this device. Please add the power

**Comment by jacob-behnken** [comment]
@danner26 any suggestion on how to do this? there are only DC screw terminals integrated on the device. I don't see any of the other IE product line templates that include a power supply for me to review examples either.

**Comment by danner26** [comment]
Something like this would do:
```
power-ports:
  - name: <NAME_OF_PSU_PORT>
    type: dc-terminal
```

You can also add `maximum_draw`, `allocated_draw` and a `label` if you have those data points/need to add a label

**Comment by jacob-behnken** [comment]
@danner26 added

**Review by danner26 — APPROVED** [review]

**Pull request merged**